Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG UBUNTU_RELEASE=jammy
+
+FROM ubuntu:$UBUNTU_RELEASE AS build-stage
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+      git ca-certificates build-essential libudev-dev libevdev-dev zlib1g-dev valac libgee-0.8-dev meson
+COPY . /
+ARG BUILD_TYPE=release
+ARG ENABLE_LTO=true
+RUN meson setup \
+      "--fatal-meson-warnings" \
+      "--buildtype=${BUILD_TYPE}" \
+      "-Db_lto=${ENABLE_LTO}" \
+      "--prefix=/release/$(uname -m)" \
```

Comment:
You should set prefix to `/usr` and pass `--destdir=/release/$(uname -m)` at install step (note that this will install into `/release/$(uname -m)/usr`). This doesn't end up mattering for my programs since they bundle all resources into the main binary itself, but would for applications that do.

Reply:
Done.